sonar-ws/src/main/java/org/sonarqube/ws/client/DefaultWsClient.java

```diff
@@ -51,17 +54,23 @@
 class DefaultWsClient implements WsClient {
 
   private final WsConnector wsConnector;
-  private final OrganizationService organizations;
-  private final PermissionsService permissionsService;
-  private final ComponentsService componentsService;
+  private final OrganizationService organizationsOld;
+  private final OrganizationsService organizations;
+  private final org.sonarqube.ws.client.permission.PermissionsService permissionsOld;
+  private final PermissionsService permissions;
+  private final org.sonarqube.ws.client.component.ComponentsService componentsOld;
+  private final ComponentsService components;
   private final FavoritesService favoritesService;
-  private final QualityProfilesService qualityProfilesService;
-  private final IssuesService issuesService;
+  private final QualityProfilesService qualityProfilesOld;
+  private final QualityprofilesService qualityprofiles;
+  private final org.sonarqube.ws.client.issue.IssuesService issuesOld;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "IssuesService"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)